src/interaction/InteractionManager.js

```diff
@@ -414,7 +412,7 @@ InteractionManager.prototype.onMouseDown = function (event)
 {
     this.mouse.originalEvent = event;
     this.eventData.data = this.mouse;
-    this.eventData.stopped = false;
+    this.eventData.stopped = event.defaultPrevented;
```

Comment:
As I've searched there is no way to check if an `Event` was stopped via `stopPropagation()`, method, and the only property that is exposed on the API is `defaultPrevented` property.

So this allow users to cancel the event via `preventDefault` method, before it enters in PIXI's InteractionManager, as the [jsfiddle](http://jsfiddle.net/endel/xnmy239h/1/) demonstrates.

What are your thoughts on this?